INSTALL.md

```diff
@@ -23,7 +23,17 @@ For Debian-like:
 
 For Fedora-like:
 
-- dnf install gcc cmake make fuse-devel polarssl-devel ruby-devel
+- dnf install gcc cmake make fuse-devel mbedtls-devel ruby-devel rubypick
+
+Alternatively, running "dnf install dislocker fuse-dislocker" to use the
+already existing RPM packages in Fedora could be a clever idea.
+
+For RHEL-like (including CentOS Scientific Linux):
+
+- yum install gcc cmake make fuse-devel mbedtls-devel ruby-devel /usr/bin/ruby
```

Comment:
Yes, if you don't know in which package e.g. the `g++` is, you can run `yum install /usr/bin/g++`. However, if the file is outside of `/bin`, `/sbin`, `/usr/bin` or `/usr/sbin`, the lookup might be slow because some Yum-related file databases need to be downloaded first (yum is doing that by itself, however these are usually large and thus it may take some time).